### PR TITLE
REF: XPub screen to TSX

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/suiyq7kwnf72fmp236pkvj4xcy?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/390)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/s7dexo5g34uncrehuygiwy3hzu?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/391)

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

File: screen/wallets/xpub.styles.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import { StyleSheet } from 'react-native';
```
Comment (Overtorment) [inline screen/wallets/xpub.styles.ts:1]: @marcosrdz 
next time use  `git mv` or however its called in your git client, so git will know that its a renamed file, and NOT deleting one and creating another

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**